src/Blockcore/EventBus/CoreEvents/BlockConnected.cs

```diff
@@ -10,9 +10,15 @@ public class BlockConnected : EventBase
     {
         public ChainedHeaderBlock ConnectedBlock { get; }
 
+        public string Hash { get; set; }
```

Comment:
This should probably be uint256

Reply:
there is a type call HashHeighttPair

Reply:
"Hash" and "Height" was just left there as it was from before. I'm removing it.

Reply:
It was previously set to string as it was only returned (Serialized) to JSON over Web Socket.